packages/components/src/docs/General/11-FAQ.mdx

```diff
@@ -78,12 +78,43 @@ import 'https://solid-design-system.fe.union-investment.de/x.x.x/components/es/a
 
 <sd-accordion>
   <h3 className="text-base sb-unstyled" slot="summary">
-    {' '}
-    Which browsers are supported?
+    What is the suggested workflow between Design and DEV in the project-specific teams? 
   </h3>
   <span className="sb-unstyled">
-    Currently the we provide auomatic VRT in Chrome and manually test our components with MS Edge, Safari and Firefox. A
-    more detailed browser matrix is in progress and will be added here soon.
+    A collaborative workflow between design and development teams is crucial for the success of any project. We recommend the following steps:
+
+    1. Initial Alignment: Begin with a kick-off meeting to ensure both designers and developers understand Solid DS and its capabilities.
+    2. Design Phase: Designers build prototypes based on Solid DS components, customising them whenever necessary while collaborating with developers to ensure feasibility. They also make all the relevant accessibility comments for later hand-off.
+    3. Hand-off Process: Designers provide detailed documentation and conduct design review sessions to ensure developers understand the specifications and rationale.
+    4. Development Phase: Maintain close communication, with regular check-ins and iterative testing to ensure designs are implemented accurately.
+    5. Final Review: Conduct a joint review to verify that the project meets design specifications and functions as intended.
+    6. Post-Launch: Keep a feedback loop for continuous improvement in future features and projects.
+
+  </span>
+</sd-accordion>
+
+<sd-accordion>
+  <h3 className="text-base sb-unstyled" slot="summary">
+    Are there any tutorials or guides available for beginners?
+  </h3>
+  <span className="sb-unstyled">
+    Yes, we offer comprehensive tutorials and guides for beginners to help them get started with the design system.
+    These resources cover from basic usage to advanced techniques, providing a smooth onboarding experience for new
+    users. Visit our <a href="https://www.figma.com/files/1075429990769806468/project/96753329">Trainings space</a> in
+    Figma to access these tutorials. Like the design system, this is a space in continuous growth and we will keep
+    adding new trainings and updating the existing ones to sharpen any edges of the overall interaction with Solid DS.
+  </span>
+</sd-accordion>
+
+<sd-accordion>
+  <h3 className="text-base sb-unstyled" slot="summary">
+    Can I download assets or code snippets directly from the documentation?
+  </h3>
+  <span className="sb-unstyled">
+    You can access and download design assets directly from our Figma documentation. This includes UI components, icons,
```

Comment:
You can access and download design assets directly from our Figma libraries. This includes UI components, icons,